example/src/screens/CustomSubtitleOnlyUI.tsx

```diff
@@ -0,0 +1,243 @@
+import React, { useCallback, useState } from 'react';
+import {
+  FlatList,
+  ListRenderItemInfo,
+  Platform,
+  StyleSheet,
+  Text,
+  TouchableOpacity,
+  View,
+} from 'react-native';
+import { useFocusEffect } from '@react-navigation/native';
+import {
+  Event,
+  PausedEvent,
+  PlayerView,
+  PlayingEvent,
+  SourceType,
+  StyleConfig,
+  SubtitleView,
+  usePlayer,
+  UserInterfaceType,
+} from 'bitmovin-player-react-native';
+import Play from '../components/icons/Play.svg';
+import Pause from '../components/icons/Pause.svg';
+import Backward from '../components/icons/Backward.svg';
+import Forward from '../components/icons/Forward.svg';
+import { useTVGestures } from '../hooks';
+
+function prettyPrint(header: string, obj: any) {
+  console.log(header, JSON.stringify(obj, null, 2));
+}
+
+interface TrackDisplay {
+  display: string;
+  identifier: string;
+}
+
+const Separator = () => <View style={styles.separator} />;
+
+const trackDisplayKeyExtractor = ({ identifier }: TrackDisplay) => identifier;
+
+export default function CustomSubtitleOnlyUI() {
+  useTVGestures();
+  const [playing, setPlaying] = useState<boolean>(false);
+
+  const styleConfig: StyleConfig =
```

Comment:
iOS has a built in subtitle only view, so we need to keep the UI enabled, but the interface type changed.